.github/workflows/release-body.yml

```diff
@@ -45,5 +45,5 @@ jobs:
         uses: softprops/action-gh-release@v1
         with:
           body_path: ./release-body.md
-          generate_release_notes: true
+          append_body: true
```

Comment:
`generate_release_notes` was not working, and btw would have loaded the diff from the latest available tag, while the releaser would use latest available `driver` tag.